### PR TITLE
Fix monsterS table

### PR DESCRIPTION
**Pull request by JonathanBraun**
Pluralized the name of our formerly singular table

### CODE REVIEW

**Comment by nwalberts** [comment]
A couple things to mention:

First: You need to silence the hound. Either via config or code. 
https://docs.google.com/document/d/1Huuzt3gIuU8gom4SScOMtEhXEqHmJ7iPKt5Yo4Rpe60/edit

Specifically having to do with string literals and quotes. You'll also need to _exclude_ you schema, but that should be linked in the default rubocop file I placed in the above doc. Let me know what you change. 

Additionally, you should only be making migrations for tables that you immediately need. Since you probably won't need reviews right away, nix that migration.

**Comment by JonathanBraun** [comment]
Made updates per Nick's comments.  (excluded Schema from rubocop.yml & removed reviews table & migration)

File: log/development.log [hunk 1]
```diff
@@ -35,3 +35,63 @@ Migrating to MonsterMigration (20170117194325)
...
+  [1m[36mActiveRecord::SchemaMigration Load (0.6ms)[0m  [1m[34mSELECT "schema_migrations".* FROM "schema_migrations"[0m
+  [1m[35m (0.2ms)[0m  [1m[34mSELECT pg_try_advisory_lock(2755645998784495215);[0m
+  [1m[36mActiveRecord::SchemaMigration Load (0.2ms)[0m  [1m[34mSELECT "schema_migrations".* FROM "schema_migrations"[0m
+Migrating to CreateReviews (20170117214438)
+  [1m[35m (0.2ms)[0m  [1m[35mBEGIN[0m
+  [1m[35m (4.4ms)[0m  [1m[35mDROP TABLE "reviews"[0m
+  [1m[35mSQL (1.0ms)[0m  [1m[31mDELETE FROM "schema_migrations" WHERE "schema_migrations"."version" = $1[0m  [["version", "20170117214438"]]
+  [1m[35m (1.4ms)[0m  [1m[35mCOMMIT[0m
+  [1m[35m (0.4ms)[0m  [1m[34mSELECT pg_advisory_unlock(2755645998784495215)[0m
+  [1m[36mActiveRecord::SchemaMigration Load (0.2ms)[0m  [1m[34mSELECT "schema_migrations".* FROM "schema_migrations"[0m
```
Comment (enthusiastick) [inline log/development.log:97]: I don't know why this file isn't being excluded via the `.gitignore` rule. Weird.

**Comment by enthusiastick** [comment]
Looks good to go!

**Pull request merged**